src/DataBuilder.php

```diff
@@ -193,10 +201,12 @@ protected function setRequestParams($config)
 
     protected function setRequestBody($config)
     {
+        
         $this->requestBody = $this->tryGet($config, 'requestBody');
         
-        if (!$this->requestBody) {
+        if (!$this->requestBody && $this->includeRawRequestBody) {
             $this->requestBody = file_get_contents("php://input");
+            $_SERVER['php://input'] = $this->requestBody;
```

Comment:
Saving request body into `$_SERVER` make sense to keep _only_ for old PHP versions due to performance reason. And even withing old PHP versions it's a huge hack, IMO, to build the logic of own application based on some workaround behavior of vendor logging library. I would recommend to just remove this line.